dusty/cli/__init__.py

```diff
@@ -96,6 +98,12 @@ def main():
     if '-d' in sys.argv:
         return run_daemon()
 
+    if '-v' in sys.argv:
```

Comment:
This can be a normal client option I think, the `-d` handling is special because we need to bypass the client docopt entirely and use the daemon docopt instead.